grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -133,7 +133,7 @@ class DocumentTemplateService {
 
         def orderItemsGroupedByProductAndUom = orderInstance?.orderItems?.findAll {
             it.orderItemStatusCode != OrderItemStatusCode.CANCELED
-        }?.groupBy { [product: it.product, unitOfMeasure: it.unitOfMeasure] }?.collect { k, v ->
+        }?.sort()?.groupBy { [product: it.product, unitOfMeasure: it.unitOfMeasure] }?.collect { k, v ->
```

Comment:
I think this should not be an issue, but wanted to be cautious. Have you checked this template for a case when the PO has only order adjustments?

Reply:
yup, it works